src/components/load-modal/section-message/section-message.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Text from '@/components/shared_ui/text';
+import { Icon } from '@/utils/tmp/dummy';
+
+type TSectionMessage = {
+    icon: string;
+    message: string;
+    className?: string;
+};
+
+const SectionMessage: React.FC<TSectionMessage> = ({ icon, message, className }) => {
+    return (
+        <div className={className}>
+            {icon && (
+                <span className='icon'>
+                    <Icon icon={icon} />
+                </span>
+            )}
```

Comment:
I think we should expect Quill icon here

Reply:
Done ✅